src/etl/permits_extractor.py

```
@@ -0,0 +1,953 @@
+                                    if not max_cursor or processed['status_date'] > max_cursor:
+                                        max_cursor = processed['status_date']
+                        
+                        logger.info(f"Processed {len(page)} records (total: {len(all_records)})")
+                        
+                        if len(page) < limit:
+                            break
+                        
+                        offset += limit
+                        
+                    except Exception as e:
+                        logger.error(f"Error fetching page at offset {offset}: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)